analyses/tp53_nf1_score/02-qc-rna_expression_score.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "QC correlation of classifier score with RNA-Seq expression"
+author: "K S Gaonkar (D3B)"
+output: html_notebook
+
+---
+    
+In this notebook we will find correlation between RNA-Seq and classifier score
+We will check if the values are inversely correlated as we would expect?
+
+### Setup
+```{r}
+library("ggpubr")
+library("ggthemes")
```

Comment:
Right now, you aren't using these. However, I do want to request scatterplots so we can see the data, as poor as the correlations may be.

Reply:
Thanks for the review! I've added the scatterplots to the notebook now.